libass/ass_bitmap.c

```diff
@@ -30,31 +30,38 @@
 #include FT_GLYPH_H
 #include FT_OUTLINE_H
 
+#include "ass_cpu.h"
 #include "ass_utils.h"
 #include "ass_outline.h"
 #include "ass_bitmap.h"
 #include "ass_render.h"
 
 
 #define ALIGN           C_ALIGN_ORDER
-#define DECORATE(func)  ass_##func##_c
+#define ENGINE_FLAGS    ASS_CPU_FLAG_NONE
```

Comment:
I think that alignment should be incorporated into cpu flags. It's also less error-prone to use something like `alignment_order(flags)` instead of hardcoded numbers.